src/main/java/org/jenkinsci/test/acceptance/po/Control.java

```diff
@@ -112,6 +112,15 @@ public void setAtOnce(String text){
         ((JavascriptExecutor)driver).executeScript("arguments[0].value = arguments[1];", e, text);
     }
 
+    /**
+     * Returns the value of the input field.
+     *
+     * @return the value of the input field.
+     */
+    public String get() {
+        WebElement e = resolve();
+        return ((JavascriptExecutor)driver).executeScript("return arguments[0].value;", e).toString();
```

Comment:
Do we need javascript for this? `WebElement#getAttribute("value")` should do. Also, note this does not work for all kinds of imputs (textarea, select, etc.)